### PR TITLE
Added new templates folder structure

### PR DESCRIPTION
**Pull request by alexander-schranz**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| License | MIT

#### What's in this PR?

Added new templates folder structure.

#### Why?

Avoid templates folder in templates.

### CODE REVIEW

File: config/templates/pages/default.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
     <key>default</key>
 
-    <view>templates/pages/default</view>
+    <view>pages/default</view>
```
Comment (alexander-schranz) [inline config/templates/pages/default.xml:8]: should we go for singular `page` as in the symfony demo? https://github.com/symfony/demo/tree/master/templates /cc @chirimoya @danrot

Comment (danrot) [reply config/templates/pages/default.xml:8]: Doesn't it feel weird to have one time singular and one plural? Would stay with it the way it is.

Comment (chirimoya) [reply config/templates/pages/default.xml:8]: would also stay with the plural version

Comment (alexander-schranz) [reply config/templates/pages/default.xml:8]: ok for me

**Pull request merged**